.travis.yml

```diff
@@ -1,11 +1,8 @@
 language: php
 php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
-  - nightly
+  - 7.2
```

Comment:
Why ignoring 7.0 and 7.1 version ? I think there is no technical reason, no ?

Reply:
I will check that, and revert it if possible :)